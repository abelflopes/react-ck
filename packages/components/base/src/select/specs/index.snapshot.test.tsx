import React from "react";
import renderer from "react-test-renderer";
import { Select } from "../index";

describe("snapshot Select", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Select />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
