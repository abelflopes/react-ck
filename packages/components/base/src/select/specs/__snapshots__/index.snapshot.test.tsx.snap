// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`snapshot Select renders correctly 1`] = `
<DocumentFragment>
  <div
    class="root skin_default standalone"
    role="button"
    tabindex="0"
  >
    <div
      class="value_slot"
    >
      <div
        class="value_content"
      >
        <span
          class="placeholder"
        />
      </div>
      <span
        class="VsR_m2r2hGIr7NfX85BU YicCROjsCroMsOGSwMAE sf93RNkSSnS5VkGCtSLh"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6.34317 7.75732L4.92896 9.17154L12 16.2426L19.0711 9.17157L17.6569 7.75735L12 13.4142L6.34317 7.75732Z"
            fill="currentColor"
          />
        </svg>
      </span>
    </div>
    <select
      class="native_element"
    >
      <option />
    </select>
    <div
      class="size_setter"
    >
      <span
        class="placeholder"
      />
    </div>
  </div>
</DocumentFragment>
`;
